### ISSUE TITLE
permissions 

### ISSUE THREAD

**Issue by ziceptor** [Arckenver/MightyLoot#11]
> >    Those are the permissions that should be given to standard players
> >    mightyloot.command
> >  mightyloot.command.hunt

mightyloot.command - this will give access to all commands. Have to explicitly prohibit other. No permission to reload.

Is it not possible to do EVERYTHING forbidden, EXCEPT those explicitly specified?

For example pointed out mightyloot.command.hunt and it is allowed. The rest is prohibited.

**Comment by Arckenver** [comment]
I don't quite understand. I've made the commands so the parent command `/ml` has the `mightyloot.command` permission and the child command `/ml hunt` has the `mightyloot.command.hunt` permission. Is it a problem ?

**Comment by ziceptor** [comment]
Allowing `mightyloot.command` for players, will give access to all permissions `mightyloot.command.*`

**Comment by ziceptor** [comment]
I'm at work. At home I'll check again.

**Comment by ziceptor** [comment]
> > Allowing mightyloot.command for players, will give access to all permissions mightyloot.command.*

To correct enough `mightyloot.command` to rename `mightyloot.command.base` or to do without him. You decide.

**Comment by Arckenver** [comment]
If you set just `mightyloot.command.hunt` for example, does it work ?

**Comment by ziceptor** [comment]
No, as it requires mightyloot.command, but if you give, there will be access to all commands.
